[OSDOCS#9670]: 4.13.34 RN (#71784)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-33-updating"]
==== Updating
To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

[id="ocp-4-13-34"]
=== RHSA-2024:0741 - {product-title} 4.13.34 bug fix and security update

Issued: 2024-02-21

{product-title} release 4.13.34, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0845[RHSA-2024:0845] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0847[RHBA-2024:0847] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.13.34 --pullspecs
----

[id="ocp-4-13-34-bug-fixes"]
==== Bug fixes

> * Previously, the release-to-channel strategy and `oc-mirror` behavior caused an error with the selective mirroring of packages.
> When you selectively mirrored the most recent (and hence default) channel for the package and a new release introduced a new channel, the current default channel became invalid and the automatic assignment of the new default channel failed.
> With this release, the issue has been resolved. 
> You can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.

Reviewer: ```suggestion
* Previously, when you selectively mirrored the most recent and default channel and a new release introduced a new channel, the current default channel became invalid. This caused the automatic assignment of the new default channel to fail. With this release, you can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.
```

Author: I don't think we need to mention the specific packages - mirroring channels was where the issue was taking place (title of bug is "oc-mirror requires that the default channel of an operator is mirrored" :) good catch!

[id="ocp-4-13-33-updating"]
==== Updating
To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].

[id="ocp-4-13-34"]

> === RHSA-2024:0741 - {product-title} 4.13.34 bug fix and security update

Reviewer 2: Looks like this still has the 4.13.33 number here:
```suggestion
=== RHSA-2024:0845 - {product-title} 4.13.34 bug fix and security update
```

Issued: 2024-02-21

{product-title} release 4.13.34, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0845[RHSA-2024:0845] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:0847[RHBA-2024:0847] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.13.34 --pullspecs
----

[id="ocp-4-13-34-bug-fixes"]
==== Bug fixes

> * Previously, when you selectively mirrored the most recent and default channel and a new release introduced a new channel, the current default channel became invalid. This caused the automatic assignment of the new default channel to fail. With this release, you can now define a `defaultChannel` field in the `ImageSetConfig` CR that overrides the `currentDefault` channel.

Reviewer 2: ```suggestion
* Previously, when you selectively mirrored the most recent and default channel and a new release introduced a new channel, the current default channel became invalid. This caused the automatic assignment of the new default channel to fail. With this release, you can now define a `defaultChannel` field in the `ImageSetConfig` custom resource (CR) that overrides the `currentDefault` channel.
```
